### DISCUSSION TITLE
Determining Test PASS/FAILURE

### DISCUSSION THREAD

**Discussion by AndyK28** [getgauge/gauge#2639] [General]
I am integrating my tests with browserstack and I want to determine if a test passed or failed after in the after scenario hook so it can report to browserstack with the status of the test. Is there a way I can go about it to fully and properly integrate my tests with browserstack? I am using c#

**Comment by AndyK28** [comment]
Does this determine whether the test passed or not?:
_context.CurrentScenario.IsFailing

**Reply by sriv** [reply]
was about to suggest that - can you use the execution context? It should set the IsFailing property.

**Reply by AndyK28** [reply]
Yes I can, I am using it in some of my hooks, thank you I'll try apply this method to determine test pass/failure
